x-pack/plugins/apm/public/components/app/service_overview/service_overview.test.tsx

```diff
@@ -45,7 +46,11 @@ function Wrapper({ children }: { children?: ReactNode }) {
       <KibanaReactContext.Provider>
         <MockApmPluginContextWrapper value={value}>
           <MockUrlParamsContextProvider
-            params={{ rangeFrom: 'now-15m', rangeTo: 'now' }}
+            params={{
+              rangeFrom: 'now-15m',
+              rangeTo: 'now',
+              latencyAggregationType: LatencyAggregationType.avg,
```

Comment:
Out of interest, why do we need this? Shouldn't it use the default value if we don't define it?

Reply:
That is because the `MockUrlParamsContextProvider` component calls `UrlParamsContext.Provider` instead of `UrlParamsProvider` which is the context the sets the default value.